controllers/serviceaccount_controller_suite_test.go

```diff
@@ -145,7 +145,7 @@ func assertRoleBinding(_ *helpers.UnitTestContextForController, ctrlClient clien
 	opts := &client.ListOptions{
 		Namespace: namespace,
 	}
-	err := ctrlClient.List(context.TODO(), &roleBindingList, opts)
+	err := ctrlClient.List(ctx, &roleBindingList, opts)
```

Comment:
Let's make ctx a parameter in the function.

Also remove the unused parameter:

```go
func assertRoleBinding(ctx context.Context, ctrlClient client.Client, namespace, name string) {
```

Reply:
Done